static/templates/case-notifications.html

```diff
@@ -0,0 +1,13 @@
+<div ng-repeat="notification in notifications" ng-switch="notification.type">
+  <div ng-switch-when="pod_action_failure">
+    <cp-alert type="danger">
+      [[ notification.payload.message ]]
+    </cp-alert>
+  </div>
+
+  <div ng-switch-when="pod_action_api_failure">
+    <cp-alert type="danger">
+      Sorry, we encountered a problem on our side while completing your action.
```

Comment:
This message is a bit bleh, but I think we're quite limited in what we can say here. We can't say we're aware of the issue, because that might not be true, depending on the deployment. We can't give a link to someone to contact, since that depends on the project/deployment (we could maybe pass it in through settings.py, but that feels like overkill for now.

Up for suggestions on something better.